src/speech_node/speech.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-#'''
 #This node listens to a service call and a topic for text to speech
```

Comment:
I like that there's a general description here.

To improve it even further: The proper way of writing a module level docstrings would be imho with triple quotes and without hashtags. This way would it would be rendered much better by automatic doc generation tools such as Sphinx:
```python
"""
This node listens to a service call and a topic for text to speech
requests. These will be processed by the festival or the Philips tts module.
"""
```